magefile.go

```diff
@@ -997,12 +997,10 @@ func (Cloud) Load() error {
 
 	// Need to get the FIPS env var flag to see if we are using the normal source cloud image name, or the FIPS variant
 	fips := os.Getenv(fipsEnv)
-	defer os.Setenv(fipsEnv, fips)
 	fipsVal, err := strconv.ParseBool(fips)
 	if err != nil {
 		fipsVal = false
 	}
-	os.Setenv(fipsEnv, strconv.FormatBool(fipsVal))
```

Comment:
Why is this removed when it was necessary before (I do not see this logic being moved to a different place in this PR)?